src/common/constants/transactions.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
```

Comment:
Wrong license snippet. Please use the one we ship with the project. If you are VSC use you can invoke it with _ninja_.

Reply:
> Wrong license snippet. Please use the one we ship with the project. If you are VSC use you can invoke it with _ninja_.

changed